examples/react-apollo/src/profile/change-email.tsx

```diff
@@ -2,15 +2,19 @@ import { useState } from 'react'
 
 import { Button, Card, Grid, TextInput, Title } from '@mantine/core'
 import { showNotification } from '@mantine/notifications'
-import { useChangeEmail, useUserEmail } from '@nhost/react'
+import { useChangeEmail, useUserEmail, useElevateSecurityKeyEmail } from '@nhost/react'
 
 export const ChangeEmail: React.FC = () => {
   const [newEmail, setNewEmail] = useState('')
+
   const email = useUserEmail()
+
   const { changeEmail } = useChangeEmail({
     redirectTo: '/profile'
   })
 
+  const { elevateEmailSecurityKey, elevated } = useElevateSecurityKeyEmail()
```

Comment:
leftover?

Reply:
Thanks, good catch. I've just removed it 👍